apps/judicial-system/web/src/routes/Prosecutor/Indictments/Processing/Processing.tsx

```diff
@@ -42,51 +40,6 @@ const Processing: React.FC = () => {
   const { setAndSendToServer } = useCase()
   const { formatMessage } = useIntl()
   const { courts } = useInstitution()
-  const { data: userData } = useQuery<{ users: User[] }>(UsersQuery, {
```

Comment:
The new components ProsecutorSection and ProsecutorSectionHeightenedSecurity are self sufficient. The get their working case from FormContext and handle all querying and workingCase modifications.